Snakefile

```diff
@@ -375,12 +392,14 @@ if config["US"].get("retrieve_US_databundle", True):
         script:
             "analysis/scripts/download_from_gdrive.py"
     
-    rule retrieve_resstock_wrmwater_heating:
+    rule retrieve_resstock_warmwater_heating:
         params:
-            gdrive_url="https://drive.google.com/drive/folders/1S-AkfE8qmjMAx3PkkI8zW0IoohTYReG3?usp=drive_link",
+            gdrive_url="https://drive.google.com/drive/folders/1Xu3774JF8MeZPuNjzGo_zxXhiZSSeQkG?usp=drive_link",
             cookies_path=pathlib.Path(".cache", "gdown"),
-            output_directory=pathlib.Path("analysis", "gdrive_data", "data"),
+            cookie_filename = "restock_warmwater_heating",
+            output_directory=pathlib.Path("analysis", "gdrive_data", "data","EnergyPlus","resstock","heating_cooling_summaries","warm_water","2018"),
```

Comment:
Mmmm.... As mentioned, it looks like we are putting too much efforts in retrieving the data. Normallly, that is a task which can be tackled by a simple call of `cp`